Content.Server/Anomaly/Effects/ChasingAnomalySystem.cs

```diff
@@ -0,0 +1,79 @@
+using System.Linq;
+using System.Numerics;
+using Content.Server.Anomaly.Components;
+using Content.Shared.Anomaly.Components;
+using Robust.Shared.Physics.Systems;
+using Robust.Shared.Random;
+
+namespace Content.Server.Anomaly.Effects;
+/// <summary>
+/// This component allows the anomaly to chase a random instance of the selected type component within a radius.
+/// </summary>
+public sealed class ChasingAnomalySystem : EntitySystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly SharedPhysicsSystem _physics = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<ChasingAnomalyComponent, AnomalyPulseEvent>(OnPulse);
+    }
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        foreach (var (anom, trans) in EntityManager.EntityQuery<ChasingAnomalyComponent, TransformComponent>(true))
```

Comment:
```
var query = EntityQueryEnumerator<ChasingAnomalyComponent, Transform>();
while (query.MoveNext(out var uid, out var anom, out var trans))
```

1. dont update paused ones
2. get uid from query not .Owner

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/c64be4e9-ac53-4f94-b554-f1b436a0f101)
@deltanedas 
I apologize for the Russian interface, ahem, but I think the error is understandable.

Reply:
my bad it should be TransformComponent not Transform i did a typo :trollface: